dashboard/app/helpers/levels_helper.rb

```diff
@@ -117,7 +117,7 @@ def select_and_remember_callouts(always_show = false)
   # Options hash for all level types
   def app_options
     # Provide the channel for templated and applab levels.
-    set_channel if @level.project_template_level || @level.game == Game.applab
+    set_channel if @level.project_template_level || @level.game == Game.applab || @level.pixelation?
```

Comment:
I've changed this line to `set_channel if @level.channel_backed?`, which I think is what Brent was suggesting.